tests/xcodeproj/BUILD.bazel

```diff
@@ -71,3 +71,13 @@ xcodeproj(
         "//tests/xcodeproj:Single-Application-UnitTests",
     ],
 )
+
+xcodeproj(
+    name = "Test-Target-With-Test-Host-Project",
+    testonly = True,
+    bazel_path = "bazelisk",
+    include_transitive_targets = False,
+    deps = [
+        "//tests/test/ios/test-imports-app:TestImports-Unit-Tests",
```

Comment:
Note that the TestImports-Unit-Tests bazel target is associated with a test host. This test target also references a symbol (named `EmptyClass`) in a framework that is linked with the test host application.

If the generated xcodeproject does *not* associated the xcode test target with the test host application, then the test run will fail at runtime with the error: ```2020-05-17 22:50:25.228654-0700 xctest[20992:5296104] The bundle “TestImports-Unit-Tests” couldn’t be loaded. Try reinstalling the bundle.
2020-05-17 22:50:25.228838-0700 xctest[20992:5296104] (dlopen(/tmp/dd/Test-Target-With-Test-Host-Project-clhcmbpljtiftpecapezszzjrnmi/Build/Products/Debug-iphonesimulator/TestImports-Unit-Tests.xctest/TestImports-Unit-Tests, 265): Symbol not found: _OBJC_CLASS_$__TtC15TestImports_App10EmptyClass
  Referenced from: /tmp/dd/Test-Target-With-Test-Host-Project-clhcmbpljtiftpecapezszzjrnmi/Build/Products/Debug-iphonesimulator/TestImports-Unit-Tests.xctest/TestImports-Unit-Tests
  Expected in: /Users/amberdixon/Downloads/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/Library/Xcode/Agents/xctest
 in /tmp/dd/Test-Target-With-Test-Host-Project-clhcmbpljtiftpecapezszzjrnmi/Build/Products/Debug-iphonesimulator/TestImports-Unit-Tests.xctest/TestImports-Unit-Tests)
Program ended with exit code: 83
```